bin/lang-js/BUCK

```diff
@@ -24,6 +24,8 @@ alias(
     actual = ":bin"
 )
 
+# we include clover here because the deno workspace
+# requires resolution of all members even if we don't use it
```

Comment:
this is out of date, you might want to rebase main

Reply:
On that note, after a rebase, you'll have another spot to update in the root BUCK file